src/main/webapp/checkout.jsp

```diff
@@ -0,0 +1,56 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<html>
+<head>
+    <title>Checkout</title>
+</head>
+<body>
+<div align="center">
+    <form action="/user/checkout" method="POST">
+        <table align="top" border="1" cellpadding="4" cellspacing="0">
+            <caption>Products list in the cart</caption>
+            <tr>
+                <th>Name</th>
+                <th>Description</th>
+                <th>Price</th>
+            </tr>
+            <c:forEach items="${user.cart.products}" var="product">
+                <tr>
+                    <td>${product.name}</td>
+                    <td>${product.description}</td>
+                    <td>${product.price}</td>
+                </tr>
+            </c:forEach>
+        </table>
+        <p>
+            <label for="name">First name:</label><br/>
+            <input type="text" name="firstName"/>
+        </p>
+        <p>
+            <label for="lastName">Last name:</label><br/>
+            <input type="text" name="lastName"/>
+        </p>
+        <p>
+            <label for="city">City:</label><br/>
+            <input type="text" name="city"/>
+        </p>
+        <p>
+            <label for="code">Street:</label></br>
```

Comment:
where we work with this data?
better create separate model - Order and write there all this information

Reply:
It will be Order in the data base. I add this entity at the next HW. Ok?

Reply:
in this case, I will assign this task together with the next hw